logger.ts

```diff
@@ -0,0 +1,39 @@
+import { createLogger, format, transports } from 'winston';
+
+const { combine, timestamp, colorize, printf } = format;
+
+const defaultFormatter = printf(({ level, message, timestamp }) => `${timestamp} [${level}] : ${message}`);
+
+export const logger = createLogger({
+  level: 'info',
+  format: combine(
+    colorize(),
+    timestamp(),
+    defaultFormatter
+  ),
+  transports: [
+    new transports.Console({
+      format: combine(
+        colorize(),
+        timestamp(),
+        defaultFormatter
+      )
+    }),
+    new transports.File({
+      filename: 'error.log',
```

Comment:
It's good to have logs in a particular folder
```suggestion
      filename: 'logs/error.log',
```